
>>> import webdb
>>> db = webdb.connect('sqlite', 'path/to/database.sqlite')
Traceback (most recent call last):
 ...
IOError: [Errno 2] No such file or directory: 'path/to/database.sqlite'
>>> db = webdb.connect('fakename')
Traceback (most recent call last):
 ...
UnknownDriver: Unable to find database driver 'fakename'
>>> db = webdb.connect('<>invalid name+/%')
Traceback (most recent call last):
 ...
UnknownDriver: Unable to find database driver '<>invalid name+/%'
>>> db = webdb.connect('base') #Module exists, but has no object named 'base'
Traceback (most recent call last):
 ...
UnknownDriver: Unable to find database driver 'base'
>>> db = webdb.connect('sqlite')
